python/tests/engine/test_python.py

```diff
@@ -599,6 +600,40 @@ def test_register_hudi_temporary_table(self):
         # Assert
         assert result is None
 
+    def test_register_hudi_temporary_table_time_travel(self):
```

Comment:
And can you add one more small test which covers the recursive thing?

Reply:
I added 2 tests in test_feature_view_engine.